content/general-docs/introduction.adoc

```diff
@@ -121,6 +121,19 @@ The rules listed in <<tab-modal-verbs>> for using modal verbs apply:
                                                  They are given for the information of the user.
 |===
 
+
+=== Typographic conventions
+
+[#tab-typographic-conventions]
+.Typographic conventions
+[%header, cols=2*]
+|===
+|Mark-up             |Definition
+|`Code elements`     |This format is used for element names of classes, material metadata types and patterns, example file names, and file endings.
```

Comment:
> material metadata types and patterns

It is used for all data types and patterns in the JSON schemas as far as I see.

Not sure if we should use it for file endings. I am open to using it for file names.

Reply:
Correct, it is used for all data types and patterins in the "Material" section. I specified the definition. 
I also removed the part about file endings.